interface/patient_file/summary/immunizations.php

```diff
@@ -822,7 +822,7 @@ function saveImmunizationObservationResults($id, $immunizationdata)
 
                                 echo "<td><input type='checkbox' class='error' id='" . attr($row["id"]) . "' value='" . xlt('Error') . "' " . $checkbox . " /></td>";
 
-                                echo "<td><button type='button' class='btn btn-danger btn-delete' id='" . attr($row["id"]) . "' value='" . xlt('Delete') . "'>" . xlt('Delete') . "</button></td>";
+                                echo "<td><button type='button' class='delete btn btn-danger btn-delete' id='" . attr($row["id"]) . "' value='" . xlt('Delete') . "'>" . xlt('Delete') . "</button></td>";
```

Comment:
This trigger class must have been removed during one of the many restyling forays. This is a bug fix so looks good to me.